app/views/articles/_form.html.erb

```diff
@@ -1,26 +1,31 @@
-<% if @article.errors.any? %>
-  <h2>The following errors prevented the article from being saved</h2>
-  <ul>
-    <% @article.errors.full_messages.each do |message| %>
-      <li><%= message %></li>
-    <% end %>
-  </ul>
-<% end %>
-
 <%# Need to provide form with the scope of the Model and the URL to submit %>
-<%# local:true specifies that we want to send via HTTP POST and not remotely via AJAX %>
+<%# local:true specifies that we want to send via HTTP POST and not remotely via
+AJAX %>
 <%# form_for comes with standard HTTP POST and won't need `local` %>
-<%= form_with(model: @article, local: true)  do |form| %>
-  <p>
-    <%= form.label :title %><br />
-    <%= form.text_field :title %>
-  </p>
-  <p>
-    <%= form.label :description %><br />
-    <%= form.text_area :description %>
-  </p>
-  <p>
-    <%= form.submit %>
-  </p>
 
-<% end %>
\ No newline at end of file
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-10">
+      <%= render 'shared/errors' %>
+      <%= form_with(model: @article, class: "shadow p-4 mb-3 bg-info rounded",
+      local: true) do |form| %>
+      <div class="form-group row">
+        <%= form.label :title, class: "col-2 col-form-label text-light" %>
+        <div class="col-10">
+          <%= form.text_field :title, class: "form-control shadow rounded" %>
+        </div>
+      </div>
+      <div class="form-group row mt-4">
+        <%= form.label :description, class: "col-2 col-form-label text-light" %>
+        <div class="col-10">
+          <%= form.text_area :description, rows: 10, class: "form-control shadow
+          rounded" %>
+        </div>
+      </div>
+      <div class="form-group row justify-content-center mt-4">
+        <%= form.submit class: "btn btn-outline-light btn-lg w-50" %>
+      </div>
```

Comment:
Are all these not part of the form tag?
Based on its indentation, it seems to mean its the same level of the form tag instead of within the form tag grouping

Reply:
Yep, you're right. I was fighting against Rubocop for that. I'll make the changes.